python/ray/_private/logging/structured_logging.py

```diff
@@ -0,0 +1,46 @@
+import logging
```

Comment:
For better organizations we can put those classes to different files. I'm thinking 

```
logging/
   constants.py
   filters.py
   formatters.py
```

Reply:
I will reorganize it, but I think dividing it into multiple files may not be necessary. The Python source code also places `class Formatter` and `class Filter` in a single file ([code](https://github.com/python/cpython/blob/main/Lib/logging/__init__.py)). We can change the structure back in the future if needed.

Reply:
Update https://github.com/ray-project/ray/pull/45098/commits/4d6769e194cac78c7dec891640f3ed849f4bc807

Reply:
I change the Python package name from "logging" to "structured_logging" because of https://github.com/ray-project/ray/pull/45098#issuecomment-2093324090.